src/flashbar/__integ__/collapsible.test.ts

```diff
@@ -129,4 +130,22 @@ describe('Collapsible Flashbar', () => {
       );
     });
   });
+
+  describe('Sticky Flashbar', () => {
+    test(
+      'keeps a space to the screen bottom to prevent the notification bar from getting cropped',
+      setupStickyFlashbarTest(async page => {
```

Comment:
[non-blocker] Would be nice to have tests on multiple screen sizes (mobile, desktop, tablet, etc).